include/picongpu/simulationControl/MySimulation.hpp

```diff
@@ -213,16 +214,26 @@ class MySimulation : public SimulationHelper<simDim>
 
             // calculate local grid points & offset
             gridSizeLocal[dim] = parserGD.getLocalSize(myGPUpos[dim]);
-            gridOffset[dim] = parserGD.getOffset(myGPUpos[dim], global_grid_size[dim]);
         }
         // by default: use an equal distributed box for all omitted params
         for (uint32_t dim = gridDistribution.size(); dim < simDim; ++dim)
         {
-            gridSizeLocal[dim] = global_grid_size[dim] / gpus[dim];
-            gridOffset[dim] = gridSizeLocal[dim] * myGPUpos[dim];
+            gridSizeLocal[dim] = globalGridSize[dim] / gpus[dim];
```

Comment:
In case it does not lead to too much changes, maybe rename one of `gridSizeLocal` and `globalGridSize` to make the names uniform?